checkov/sca_package/output.py

```diff
@@ -50,7 +50,67 @@ def output_row(self) -> List[str]:
         ]
 
 
-def create_report_record(
+def create_report_license_record(
+    rootless_file_path: str,
+    file_abs_path: str,
+    check_class: str,
+    licenses_status: _LicenseStatus,
+    runner_filter: RunnerFilter | None = None,
+) -> Record:
+    runner_filter = runner_filter or RunnerFilter()
+    package_name = licenses_status["package_name"]
+    package_version = licenses_status["package_version"]
+
+    check_result: _CheckResult = {
+        "result": CheckResult.FAILED,
+    }
+
+    policy = licenses_status["policy"]
+    status = licenses_status["status"]
+
+    # to-do: check whether we want to save records for passed "COMPLIANT" licenses
+    if status == "COMPLIANT":
+        check_result = {
+            "result": CheckResult.PASSED,
+        }
+
+    # to-do: ask if it is right to put it here
+    # if runner_filter.skip_cve_package and package_name in runner_filter.skip_cve_package:
+    #     check_result = {
+    #         "result": CheckResult.SKIPPED,
+    #         "suppress_comment": f"Filtered by package '{package_name}'"
+    #     }
+
+    code_block = [(0, f"{package_name}: {package_version}")]
+
+
+
+    details = {
+        "package_name": package_name,
+        "package_version": package_version,
+        "license": licenses_status["license"],
+        "status": status,
+        "policy": policy,
+    }
+
+    record = Record(
+        check_id="",
+        bc_check_id=policy,
+        check_name="SCA license scan",
```

Comment:
please create a constant, because you will probably check for it later 😄

Reply:
sure! will be fixed